08 Drafts/03 Writing Algorithms/12 Universes/03 Equity/02 Fundamental Universes/06 Examples.html

```diff
@@ -35,7 +35,7 @@ <h4>Example 2: Take 10 stocks above their 200-Day EMA and have more than $1B dai
     return (from f in fundamental
         let avg = _stateData.GetOrAdd(f.Symbol, sym =&gt; new SelectionData(200))
         where avg.Update(f.EndTime, f.AdjustedPrice)
-        where f.Price &gt; avg.Ema &amp;&amp; f.DollarVolume &gt; 1000000000
+        where f.Price &gt; avg.Ema.Current.Value &amp;&amp; f.DollarVolume &gt; 1000000000
```

Comment:
We don't need `.Current.Value` in C#, so let's remove it.
I see it gives us different code from C# and Python, but if C# has sugar to make the code shorter, we should use it.